cognite/client/_api/datapoints.py

```diff
@@ -411,7 +417,128 @@ def retrieve_dataframe(
                 granularity=granularity,
                 limit=limit,
             ).to_pandas()
-        return pd.concat([id_df, external_id_df], axis="columns")
+        df = pd.concat([id_df, external_id_df], axis="columns")
+        complete = complete or ""
```

Comment:
maybe `complete = [s.strip() for s in complete.split(",")] if complete else []` is better. Then it won't matter if the user passes "fill,dropna", "fill, dropna", or "dropna, fill" etc.

Reply:
currently works anyway since it checks substring by in, but probably list is cleaner/safer. added this and made the value check stronger